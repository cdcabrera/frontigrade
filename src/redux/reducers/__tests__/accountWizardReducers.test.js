import { accountWizardReducers } from '../';

describe('ToastNotificationsReducers', () => {
  it('should return the initial state', () => {
    expect(accountWizardReducers.initialState).toBeDefined();
  });
});
