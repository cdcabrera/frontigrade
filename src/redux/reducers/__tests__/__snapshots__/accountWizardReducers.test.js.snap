// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fulfilled types 1`] = `
Object {
  "result": Object {
    "account": Object {},
    "add": false,
    "configuration": Object {},
    "edit": false,
    "error": false,
    "errorMessage": null,
    "fulfilled": false,
    "show": false,
    "stepPolicyValid": false,
    "stepRoleValid": true,
    "stepThreeValid": false,
  },
  "type": "ADD_ACCOUNT",
}
`;

exports[`fulfilled types 2`] = `
Object {
  "result": Object {
    "account": Object {},
    "add": false,
    "configuration": Object {},
    "edit": false,
    "error": false,
    "errorMessage": null,
    "fulfilled": false,
    "show": false,
    "stepPolicyValid": false,
    "stepRoleValid": true,
    "stepThreeValid": false,
  },
  "type": "DELETE_ACCOUNT",
}
`;

exports[`fulfilled types 3`] = `
Object {
  "result": Object {
    "account": Object {},
    "add": false,
    "configuration": Object {},
    "edit": false,
    "error": false,
    "errorMessage": null,
    "fulfilled": false,
    "show": false,
    "stepPolicyValid": false,
    "stepRoleValid": true,
    "stepThreeValid": false,
  },
  "type": "DELETE_ACCOUNTS",
}
`;

exports[`fulfilled types 4`] = `
Object {
  "result": Object {
    "account": Object {},
    "add": false,
    "configuration": Object {},
    "edit": false,
    "error": false,
    "errorMessage": null,
    "fulfilled": false,
    "show": false,
    "stepPolicyValid": false,
    "stepRoleValid": true,
    "stepThreeValid": false,
  },
  "type": "GET_ACCOUNT",
}
`;

exports[`fulfilled types 5`] = `
Object {
  "result": Object {
    "account": Object {},
    "add": false,
    "configuration": Object {},
    "edit": false,
    "error": false,
    "errorMessage": null,
    "fulfilled": false,
    "show": false,
    "stepPolicyValid": false,
    "stepRoleValid": true,
    "stepThreeValid": false,
  },
  "type": "GET_ACCOUNTS",
}
`;

exports[`fulfilled types 6`] = `
Object {
  "result": Object {
    "account": Object {},
    "add": false,
    "configuration": Object {},
    "edit": false,
    "error": false,
    "errorMessage": null,
    "fulfilled": false,
    "show": false,
    "stepPolicyValid": false,
    "stepRoleValid": true,
    "stepThreeValid": false,
  },
  "type": "ADD_ACCOUNT_WIZARD",
}
`;

exports[`fulfilled types 7`] = `
Object {
  "result": Object {
    "account": Object {
      "test": "success",
    },
    "add": false,
    "configuration": Object {},
    "edit": false,
    "error": false,
    "errorMessage": null,
    "fulfilled": false,
    "show": false,
    "stepPolicyValid": true,
    "stepRoleValid": true,
    "stepThreeValid": false,
  },
  "type": "ADD_ACCOUNT_WIZARD_STEP_POLICY",
}
`;

exports[`fulfilled types 8`] = `
Object {
  "result": Object {
    "account": Object {
      "test": "success",
    },
    "add": false,
    "configuration": Object {},
    "edit": false,
    "error": false,
    "errorMessage": null,
    "fulfilled": false,
    "show": false,
    "stepPolicyValid": false,
    "stepRoleValid": true,
    "stepThreeValid": false,
  },
  "type": "INVALID_ACCOUNT_WIZARD_STEP_POLICY",
}
`;

exports[`fulfilled types 9`] = `
Object {
  "result": Object {
    "account": Object {},
    "add": false,
    "configuration": Object {},
    "edit": false,
    "error": false,
    "errorMessage": null,
    "fulfilled": false,
    "show": false,
    "stepPolicyValid": false,
    "stepRoleValid": true,
    "stepThreeValid": false,
  },
  "type": "ADD_ACCOUNT_WIZARD_STEP_ROLE",
}
`;

exports[`fulfilled types 10`] = `
Object {
  "result": Object {
    "account": Object {},
    "add": false,
    "configuration": Object {},
    "edit": false,
    "error": false,
    "errorMessage": null,
    "fulfilled": false,
    "show": false,
    "stepPolicyValid": false,
    "stepRoleValid": true,
    "stepThreeValid": false,
  },
  "type": "INVALID_ACCOUNT_WIZARD_STEP_ROLE",
}
`;

exports[`fulfilled types 11`] = `
Object {
  "result": Object {
    "account": Object {},
    "add": false,
    "configuration": Object {},
    "edit": false,
    "error": false,
    "errorMessage": null,
    "fulfilled": false,
    "show": false,
    "stepPolicyValid": false,
    "stepRoleValid": true,
    "stepThreeValid": false,
  },
  "type": "ADD_ACCOUNT_WIZARD_STEPTHREE",
}
`;

exports[`fulfilled types 12`] = `
Object {
  "result": Object {
    "account": Object {},
    "add": false,
    "configuration": Object {},
    "edit": false,
    "error": false,
    "errorMessage": null,
    "fulfilled": false,
    "show": false,
    "stepPolicyValid": false,
    "stepRoleValid": true,
    "stepThreeValid": false,
  },
  "type": "INVALID_ACCOUNT_WIZARD_STEPTHREE",
}
`;

exports[`fulfilled types 13`] = `
Object {
  "result": Object {
    "account": Object {},
    "add": true,
    "configuration": Object {},
    "edit": false,
    "error": false,
    "errorMessage": null,
    "fulfilled": false,
    "show": true,
    "stepPolicyValid": false,
    "stepRoleValid": true,
    "stepThreeValid": false,
  },
  "type": "ADD_ACCOUNT_SHOW",
}
`;

exports[`fulfilled types 14`] = `
Object {
  "result": Object {
    "account": Object {},
    "add": false,
    "configuration": Object {},
    "edit": false,
    "error": false,
    "errorMessage": null,
    "fulfilled": false,
    "show": false,
    "stepPolicyValid": false,
    "stepRoleValid": true,
    "stepThreeValid": false,
  },
  "type": "EDIT_ACCOUNT_SHOW",
}
`;

exports[`fulfilled types 15`] = `
Object {
  "result": Object {
    "account": Object {},
    "add": false,
    "configuration": Object {},
    "edit": false,
    "error": false,
    "errorMessage": null,
    "fulfilled": false,
    "show": false,
    "stepPolicyValid": false,
    "stepRoleValid": true,
    "stepThreeValid": false,
  },
  "type": "UPDATE_ACCOUNT_HIDE",
}
`;
